python/example/single_cell_multicompartment.py

```diff
@@ -23,32 +24,42 @@
 
 # Start with a spherical soma with radius 6 μm,
 # approximated with a cylinder of: length = diameter = 12 μm.
-s  = b.add_cable(length=12, radius=6, name="soma", ncomp=1)
+
+s = tree.append(arbor.mnpos, arbor.mpoint(-12, 0, 0, 6), arbor.mpoint(0, 0, 0, 6), tag=1)
 
 # Add the dendrite cables, labelling those closest to the soma "dendn",
 # and those furthest with "dendx" because we will set different electrical
 # properties for the two regions.
-b0 = b.add_cable(parent=s,  length=100, radius=2, name="dendn", ncomp=100)
+
+labels = arbor.label_dict()
+labels['soma'] = '(tag 1)'
```

Comment:
I would also give different tags for `dendn` and `dendx` something >= 5 so as not to interfere with the "reserved" SWC tags

Reply:
I think that this might be ok: Arbor explicitly lets you define tags how you please.

Reply:
We aren't calling these things SWC samples or using any SWC loaders so you're right that it doesn't matter here. I've been using too much SWC. But it might be better to stick to the convention regardless.

Reply:
Done